transcendence-app/src/user/user.service.ts

```diff
@@ -84,6 +84,8 @@ export class UserService {
         avatarId,
         avatarX: 0,
         avatarY: 0,
+        twoFactorAuthenticationSecret: null,
```

Comment:
Is there some case where we could have `twoFactorAuthenticationSecret === null`, and `isTwoFactorAuthenticationEnabled === true` ?

Reply:
No, but there is a case where `twoFactorAuthenticationSecret !== null` and `isTwoFactorAuthenticationEnabled === false` after the user generates a QR code and before they enable the 2FA.